element-connector/src/main/java/org/eclipse/californium/elements/tcp/CloseOnErrorHandler.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Institute for Pervasive Computing, ETH Zurich and others.
```

Comment:
In classes you created from scratch you may as well just indicate Amazon as the copyright holder (we at Bosch do, at least) ... but that is up to you, I guess.